internal/pkg/agent/install/uninstall.go

```diff
@@ -100,6 +105,27 @@ func Uninstall(cfgFile, topPath, uninstallToken string, log *logp.Logger, pt *pr
 		}
 	}
 
+	// will only notify fleet of the uninstall command if it can gather config and agentinfo, and is not a stand-alone install
+	notifyFleet := false
+	var ai *info.AgentInfo
+	c, err := operations.LoadFullAgentConfig(ctx, log, cfgFile, false, unprivileged)
+	if err != nil {
+		pt.Describe(fmt.Sprintf("unable to read agent config to deterimine if notifiying fleet-server is needed: %v", err))
```

Comment:
Not sure if a user needs this level of detail when uninstalling with the actual error displayed on the progress bar :thinking: 
Perhaps a more generic message about being unable to notify Fleet (only if it's a managed agent) of the agent removal would be more appropriate and the rest of the errors can be just logged